packages/riipen-ui/src/components/Button.jsx

```diff
@@ -322,11 +317,20 @@ class Button extends React.Component {
 
           .outlined {
             background-color: transparent;
-            border: 1px solid rgba(0, 0, 0, 0.23);
             color: ${theme.palette.text.primary};
             overflow: hidden;
             position: relative;
           }
+          .outlined::after {
+            border-radius: ${theme.shape.borderRadius.md};
+            border: 1px solid rgba(0, 0, 0, 0.23);
+            bottom: 0;
+            content: "";
+            left: 0;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
```

Comment:
Does this just make the border not take up the extra pixel?

Reply:
Yeah, it creates a pseudo-element and then applies the border to that, so it doesn't take up the extra pixel